### ISSUE TITLE
Error on tauri project

### ISSUE THREAD

**Issue by Anivie** [intellij-rust/intellij-rust#10859]
## Environment

* **IntelliJ Rust plugin version:** 0.4.200.5421-232
* **Rust toolchain version:** 1.72.0 (5680fa18f 2023-08-23) x86_64-pc-windows-msvc
* **IDE name and version:** CLion 2023.2.1 (CL-232.9559.58)
* **Operating system:** Windows 11.0
* **Macro expansion:** enabled
* **Additional experimental features:** org.rust.macros.proc.attr

## Problem description
Wrong code tip: Clion reports that there is an error in the code, but in fact the code is correct
![image](https://github.com/intellij-rust/intellij-rust/assets/88354708/e0b04095-ad8f-471b-9b47-1826481edeb6)

## Steps to reproduce
Use `cargo create-tauri-app` -> Use TS as frontend -> pnpm -> Vue
Import project to Clion

**Comment by Maksim20023** [comment]
Hi! Thank you for the report.
I reproduce this issue, but this warning is observed only if the external linter is turned on.
Could you please specify if you use an external linter?
If I understand correctly, this warning is related to the peculiarities of the project checking performed in the IDE. Please take a look at this discussion on the Tauri issue tracker: https://github.com/tauri-apps/tauri/issues/3142
As a workaround to avoid warning, you can create 'dist' directory in the project's root. Please try to do it and write a response, whether it helped.

**Comment by Anivie** [comment]
Thanks for your help, this error report disappeared after I created the directory.
However, I found that this problem did not occur in the following scenarios:
- Run it manually in the project directory`cargo check`
```
PS C:\Users\anivi\OneDrive\Document\Desktop\tauri-app\src-tauri> cargo check
    Checking tauri v1.4.1
    Checking tauri-app v0.0.0 (C:\Users\anivi\OneDrive\文档\桌面\tauri-app\src-tauri)
    Finished dev [unoptimized + debuginfo] target(s) in 2.22s
```

 - Import the project in VSCode

**Comment by dbidwell94** [comment]
Hey, OP :) This is not an error with RustRover. This is because you have not yet generated your `dist` folder. Do so by running a build on the TS/JS side of the application. For example: `npm run build` instead of `npm run tauri build`
